### DISCUSSION TITLE
How do I specify a pre-existing Julia installation when using JuliaCall?

### DISCUSSION THREAD

**Discussion by pscarpati** [cjdoris/PythonCall.jl#514] [Q&A]
I noticed that when pip installs JuliaCall, it also creates a julia installation inside of .julia
is it possible to skip this step and link JuliaCall to a specific Julia installation that already exists?

Many thanks

**Comment by cjdoris** [comment]
Yep:

https://juliapy.github.io/PythonCall.jl/stable/juliacall/#julia-deps

https://github.com/JuliaPy/PyJuliaPkg?tab=readme-ov-file#configuration
